v3/plugins/bitlang/cobol/5.0.6/meta.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v2
+publisher: bitlang
+name: cobol
+version: 5.0.6
+type: VS Code extension
+displayName: COBOL
+title: COBOL
+description: COBOL, JCL, PL/I & DIR syntax support
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/spgennard/vscode_cobol
+category: Language
+spec:
+  extensions:
+    - https://github.com/eclipse/che-che4z/releases/download/1.0.0/cobol-5.0.6.vsix
```

Comment:
Please ensure all files end in newlines.

Reply:
Added missing newlines